ext/lmdb-safe/lmdb-safe.hh

```diff
@@ -87,7 +87,7 @@ private:
   std::map<std::thread::id, int> d_ROtransactionsOut;
 };
 
-std::shared_ptr<MDBEnv> getMDBEnv(const char* fname, int flags, int mode);
+std::shared_ptr<MDBEnv> getMDBEnv(const char* fname, int flags, int mode, uint64_t mapsizeMB=(sizeof(long)==4) ? 100 : 16000);
```

Comment:
LLP64 system do exists (ew.g. WIN64). Although I don't think we support them, I still prefer
```suggestion
std::shared_ptr<MDBEnv> getMDBEnv(const char* fname, int flags, int mode, uint64_t mapsizeMB=(sizeof(void *)==4) ? 100 : 16000);
```